projects/framework-blender/pyproject.toml

```diff
@@ -0,0 +1,75 @@
+[tool.poetry]
+name = "ftrack-framework-blender"
+version = "24.6.0"
```

Comment:
This should be updated to 24.10.0rc1 or similar.

Reply:
I'll go for 24.11.0rc1